src/components/editor/inputs/InputLang.jsx

```diff
@@ -96,12 +99,10 @@ const InputLang = (props) => {
               <input
                 type="radio"
                 className="form-check-input"
-                name="lang"
-                id={`noLangRadio-${props.textValue}`}
-                data-testid={`noLangRadio-${props.textValue}`}
+                id="absent"
```

Comment:
1. get rid of the `name="lang"` to prevent the browser from trying to set the value of the radio button based on the grouping (we set based on our value in state)
2. i don't think we need an `id` or `data-testid` here, just get rid of for simplicity